sharding/nodesSetup.go

```diff
@@ -201,20 +224,20 @@ func (ns *NodesSetup) InitialNodesPubKeys() map[uint32][]string {
 }
 
 // InitialNodesInfo - gets initial nodes info
-func (ns *NodesSetup) InitialNodesInfo() map[uint32][]*NodeInfo {
-	return ns.allNodesInfo
+func (ns *NodesSetup) InitialNodesInfo() (map[uint32][]*NodeInfo, map[uint32][]*NodeInfo) {
+	return ns.eligible, ns.waiting
 }
 
 // InitialNodesPubKeysForShard - gets initial nodes public keys for shard
 func (ns *NodesSetup) InitialNodesPubKeysForShard(shardId uint32) ([]string, error) {
```

Comment:
This method only returns the public keys of the eligible Nodes, maybe it should be renamed to `InitialEligibleNodesPubKeysForShard()`.

Reply:
done